apps/myaccount/src/pages/personal-info.tsx

```diff
@@ -52,7 +52,10 @@ const PersonalInfoPage = (): ReactElement => {
             pageDescription={ isFeatureEnabled(accessConfig?.personalInfo,
```

Comment:
Shall we reformat the logic for better readability

```suggestion
            pageDescription={
                isFeatureEnabled(...)
                    ? t("myAccount:pages.personalInfo.subTitle")
                    : isFeatureEnabled(...)
                        ? t("myAccount:pages.personalInfoWithoutLinkedAccounts.subTitle")
                        : t("myAccount:pages.personalInfoWithoutExportProfile.subTitle")
           }             
```

Reply:
Resolved with 479bbf6